### PR TITLE
added test

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: lib/query.dart [hunk 1]
```diff
@@ -1,61 +1,117 @@
...
+  /// the query will return resources where [attribute] is equal
+  /// to any of the values in the list.
   static String notEqual(String attribute, dynamic value) =>
       _addQuery(attribute, 'notEqual', value);
 
+  /// Filter resources where [attribute] is less than [value].
+  ///
+  /// [value] can be a single value or a list. If a list is used
+  /// the query will return resources where [attribute] is equal
+  /// to any of the values in the list.
```
Comment (Meldiron) [inline lib/query.dart:27]: I think lessThan, greaterThan, (and equal variants) only support 1 value, not array.

File: lib/query.dart [hunk 2]
```diff
@@ -1,61 +1,117 @@
...
   static String greaterThanEqual(String attribute, dynamic value) =>
       _addQuery(attribute, 'greaterThanEqual', value);
 
+  /// Filter resources where by searching [attribute] for [value].
+  /// 
+  /// A fulltext index on [attribute] is required for this query to work.
```
Comment (Meldiron) [inline lib/query.dart:57]: I think it's not required since 1.3. Let's not mention fulltext index.

File: pubspec.yaml [hunk 3]
```diff
@@ -18,17 +18,18 @@ environment:
...
-  web_socket_channel: ^2.3.0
-  universal_html: ^2.0.9
+  cookie_jar: ^4.0.3
+  device_info_plus: ^9.0.2
+  flutter_web_auth_2: ^2.1.4
+  http: '>=0.13.6 <1.0.1'
+  package_info_plus: ^4.0.2
+  path_provider: ^2.0.15
+  web_socket_channel: ^2.4.0
+  universal_html: ^2.2.2
```
Comment (Meldiron) [inline pubspec.yaml:28]: Possible breakign change for cookie_jar, device_info_plus, package_info_plus,

Comment (lohanidamodar) [reply pubspec.yaml:28]: working.

File: lib/query.dart (hunk 2 above)
Comment (lohanidamodar) [reply lib/query.dart]: updated

File: lib/query.dart (hunk 1 above)
Comment (lohanidamodar) [reply lib/query.dart]: updated

**Pull request merged**